tests/e2e/weaviate_e2e/utils.py

```diff
@@ -0,0 +1,388 @@
+def get_query_vector():
```

Comment:
I'm thinking whether this should be a constant instead as we don't expect to mutate this vector

Reply:
Yeah, makes sense.